numpy/core/tests/test_umath.py

```diff
@@ -1840,6 +1848,23 @@ def test_reduction(self):
             btype = np.array([True], dtype=object)
             assert_(type(f.reduce(btype)) is bool, msg)
 
+    @pytest.mark.parametrize("input_dtype_obj, bitsize",
+            zip(bitwise_types, bitwise_bits))
+    def test_popcount(self, input_dtype_obj, bitsize):
```

Comment:
Adding a test for negative signed integers might be a good idea.
I believe those involve taking their absolute value, correct? Or do they raise?

Reply:
We do an `abs()` yes. Will add it

Reply:
`np.iinfo(np.int64).min` (the lowest value) is a bit interesting, since `-int_min == int_min` (abs does nothing).  But I think that actually just works out here, because casting to uint will give the right result.